presto-main/src/main/java/com/facebook/presto/metadata/FunctionRegistry.java

```diff
@@ -473,7 +473,8 @@ public FunctionRegistry(TypeManager typeManager, BlockEncodingSerde blockEncodin
                 .scalars(SessionFunctions.class)
                 .scalars(StringFunctions.class)
                 .scalars(WordStemFunction.class)
-                .scalar(SplitToMapFunction.class)
+                .scalar(SplitToMapFunction.ResolveDuplicateKeysFunction.class)
```

Comment:
I'll just call them `SplitToMapFunction.ResolveDuplicateKeys` and `SplitToMapFunction.FailOnDuplicateKeys`.

Reply:
fixed